sdk/src/main/scala/com/horizen/validation/ConsensusValidator.scala

```diff
@@ -46,7 +46,7 @@ class ConsensusValidator extends HistoryBlockValidator with ScorexLogging {
 
     val lastBlockInPreviousConsensusEpochInfo: SidechainBlockInfo = history.blockInfoById(history.getLastBlockInPreviousConsensusEpoch(verifiedBlock.timestamp, verifiedBlock.parentId))
     val previousFullConsensusEpochInfo: FullConsensusEpochInfo = history.getFullConsensusEpochInfoForBlock(lastBlockInPreviousConsensusEpochInfo.timestamp, lastBlockInPreviousConsensusEpochInfo.parentId)
-    verifyOmmers(verifiedBlock, currentConsensusEpochInfo, previousFullConsensusEpochInfo, verifiedBlock.parentId, parentBlockInfo, history, Seq())
+    verifyOmmers(verifiedBlock, currentConsensusEpochInfo, Some(previousFullConsensusEpochInfo), verifiedBlock.parentId, parentBlockInfo, history, Seq())
```

Comment:
btw better use Option() instead of Some(), because in first case we get correct auto type Option, but not Some.